src/garage/torch/algos/vpg.py

```diff
@@ -275,9 +331,16 @@ def _compute_objective(self, advantages, valids, obs, actions, rewards):
             torch.Tensor: Calculated objective values
 
         """
-        # pylint: disable=unused-argument
+        del rewards
```

Comment:
if you don't need it, why keep it as an argument?

Reply:
In this case, the `rewards` parameter is used in PPO